### DISCUSSION TITLE
A couple questions

### DISCUSSION THREAD

**Discussion by Croworbit** [Tyrrrz/YoutubeDownloader#365] [Q&A] [answered]
what does the program mean when it says 'inject media tags'? also, apparently downloaded videos have 'an additional stream' which I have no idea what means

**Comment by Tyrrrz** [comment]
Inject media tags -- mp3 files will have artist/title/album information, if it can be pulled from MusicBrainz.

Additional stream -- not sure, you'd have to be more specific.

**Comment by Croworbit** [comment]
![vcs](https://github.com/Tyrrrz/YoutubeDownloader/assets/138729134/cbe8694f-09e9-42ed-b9b8-53f0a38be8cc)
this is what my video editor says when I pull in videos, I'm not too sure what it means

**Reply by Tyrrrz** [reply, marked as answer]
Yeah, it's hard to say based on that. Each video should have 1 or 2 streams (audio, and optionally video) and a varying number of subtitle tracks (which are sometimes referred to as streams as well).

**Reply by Croworbit** [reply]
if I try interacting with the additional stream, it seems just like a still image and no sound, the same length of the video
I'll assume it is a subtitle track, are there any options to exclude these extra streams, or to view which the video has?

**Reply by Tyrrrz** [reply]
You should be able to exclude them when importing to whatever software you're using.

**Reply by Croworbit** [reply]
you're probably right - I just hope these extra streams it downloads aren't costing me time/affecting my internet cap.
